deploy/operator.yaml

```diff
@@ -35,4 +35,4 @@ spec:
                 fieldRef:
                   fieldPath: metadata.labels['name']
             - name: DEBUG
-              value: "false"
+              value: "true"
```

Comment:
Just changing here will make the CSV updated.

Reply:
done